api/mapping/timeseries/bundle.md

```diff
@@ -62,6 +62,21 @@ After registering the new type, TimeseriesLayerService will create delegates aut
 No configuration is required.
 
 
+## Bundle state
+
+```json
+{
+  "ui": "player|range|none",
+  "metadata": {
+    "layer": "<metadata layer id>",
+    "attribute": "<time attribute used to filter metadata layer>",
+    "toggleLevel": "<map toggle level at which the metadata layer will be shown>"
+  }
+}
+```
+
+The metadata is in use when `range` ui is selected
```

Comment:
This is the configuration that can be added to a layer. The bundle state is something like:
```
{
    "time": "<time in some format or an array(?)"
}
```

Reply:
And that state is only recognized by the "range" UI right?

Reply:
Currently yes